src/app/static/src/app/components/plots/MapAdjustScale.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-    <div class="p-3 ml-2">
+    <div class="p-3">
         <div class="form-check dynamic-filtered">
             <label class="form-check-label">
                 <input id="type-input-dynamic-filtered" class="form-check-input" type="radio" :name="scaleTypeGroup"
                        :value="scaleType.DynamicFiltered"
                        :checked="selectedScale.type === scaleType.DynamicFiltered"
-                       @change="updateOutputColourScale(indicatorMetadata.indicator, {...selectedScale, type: parseInt($event.target.value)})">
+                       @change="updateOutputScale(name, {...selectedScale, type: parseInt($event.target.value)})">
```

Comment:
Was this change just to make things a little less verbose..?

Reply:
And because this now updates the size or the colour scale depending on the "name" prop.